FastSimulation/TrackerSetup/plugins/TrackerInteractionGeometryESProducer.cc

```diff
@@ -1,26 +1,39 @@
-#include "FastSimulation/TrackerSetup/plugins/TrackerInteractionGeometryESProducer.h"
-#include "RecoTracker/Record/interface/TrackerRecoGeometryRecord.h"
-
 #include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ESProducer.h"
 #include "FWCore/Framework/interface/ModuleFactory.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FastSimulation/TrackerSetup/interface/TrackerInteractionGeometry.h"
+#include "FastSimulation/TrackerSetup/interface/TrackerInteractionGeometryRecord.h"
+#include "RecoTracker/Record/interface/TrackerRecoGeometryRecord.h"
 
 #include <memory>
+#include <string>
+
+class TrackerInteractionGeometryESProducer : public edm::ESProducer {
+public:
+  TrackerInteractionGeometryESProducer(const edm::ParameterSet& p);
+  ~TrackerInteractionGeometryESProducer() override;
+  std::unique_ptr<TrackerInteractionGeometry> produce(const TrackerInteractionGeometryRecord&);
+
+private:
+  edm::ESGetToken<GeometricSearchTracker, TrackerRecoGeometryRecord> geoSearchToken;
+  std::string _label;
+  edm::ParameterSet theTrackerMaterial;
```

Comment:
```suggestion
  edm::ESGetToken<GeometricSearchTracker, TrackerRecoGeometryRecord> geoSearchToken_;
  std::string label_;
  edm::ParameterSet theTrackerMaterial_;
```
I'd avoid leading unedrscores in private class members. Since you are there, you can go forward and use the same convention (trailing underscore) for all class members